client/src/helpers/qnAHelper.js

```diff
@@ -63,89 +63,143 @@ module.exports = {
     return newArr;
 
   },
+//HOLY MOLY - this function got crazy - needed to refactor because it was duplicating answers displayed after sort
+//now it seems to function properly.  Sorts answers by seller and sellerhelpfulness, then sorts remaining user answers
+//based on helpfulness
+  sortAnswers(answers) {
+    //create new storage array
+    let newArr = new Array(answers.length);
+    //handle questions without answers (empty answerArr)
+    answers.forEach((answerArr, index) => {
+      if (!answerArr.length) {
+        newArr[index] = answerArr
+      }
+      //if there are answers
+      if (answerArr.length) {
```

Comment:
might not need this second if as it seems like the first if statement will result in anything going through having length

Reply:
🔥 ty, ill take a look!  good eye